data-access/Helpers/Exceptions.cs

```diff
@@ -123,59 +123,59 @@ public class NoPermissions : ForbiddenDataAccessException
 		{
 			private const string customMsg = "User $UserId has no permissions associated with edition $EditionId.";
 
-			public NoPermissions(UserInfo user)
+			public NoPermissions(EditionUserInfo editionUser)
```

Comment:
I've obviously missed that in a previous pull request - exception classes usually end with Exception, so `NoPermissionsException`, `NoReadPermissionException`, etc..

Reply:
Nice catch! I will go through and refactor these.